tools/gulp/tasks/lint.ts

```diff
@@ -46,6 +54,56 @@ task('madge', ['material:clean-build'], () => {
   });
 });
 
+task('ownerslint', () => {
+  let errors = 0;
+
+  let ownedPaths = readFileSync(ownersFilePath, 'utf8').split('\n')
+      // Trim lines.
+      .map(line => line.trim())
+      // Remove empty lines and comments.
+      .filter(line => line && !line.startsWith('#'))
+      // Split off just the path glob and turn it into a regex.
+      .map(line => new Minimatch(line.split(/\s+/)[0], {dot: true, matchBase: true}))
```

Comment:
You can avoid having to repeat the `Minimatch` constructor config by `concat`-ing the `ignoreOwners` earlier:

```ts
readFileSync(ownersFilePath, 'utf8').split('\n')
  // Trim lines.
  .map(line => line.trim())
  // Remove empty lines and comments.
  .filter(line => line && !line.startsWith('#'))
  // Take out the file pattern.
  .map(line => line.split(/\s+/)[0])
  // Add in the paths we're ignoring.
  .concat(ignoreOwners)
  // Turn the paths into regex.
  .map(line => new Minimatch(line, {dot: true, matchBase: true}))
```